src/Mod/Spreadsheet/TestSpreadsheet.py

```diff
@@ -894,7 +894,7 @@ def plm_equal(plm1, plm2):
         pla = FreeCAD.Placement(vec,rot)
         ipla = pla.inverse()
 
-        sheet.set('A1', '=create(<<vector>>, 2, 1, 2)')
+        sheet.set('A1', '=vector(2, 1, 2)')
```

Comment:
Since the old syntax should continue to work, the tests shouldn't remove it, but just add to it.

Reply:
Thanks for that, I added a few tests which verify the old usage as well.